cypress/support/pageBuilder/pbDeleteBlocks.ts

```diff
@@ -0,0 +1,42 @@
+import { GraphQLClient } from "graphql-request";
+
+const mutation = /* GraphQL */ `
+    mutation DeletePageBlock($id: ID!) {
+        pageBuilder {
+            deletePageBlock(id: $id) {
+                error {
+                    code
+                    message
+                    __typename
+                }
+                __typename
+            }
+        }
+    }
+`;
+
+Cypress.Commands.add("pbDeleteBlocks", () => {
```

Comment:
```suggestion
Cypress.Commands.add("pbDeleteAllBlocks", () => {
```

This deletes all blocks, so let's have the name of the command reflect that.